teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'unknown')
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] To avoid errors, kindly include the `parent_permission` in your definition, like so:

  ```
  def initialize(specialization, age, name = 'unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @specialization = specialization
  end